Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -45,7 +45,7 @@ - (void)setupNavigationController
     vc.filter = self.filter;
     vc.allowMultipleSelection = self.allowMultipleSelection;
     if (!self.dataSource) {
-        self.dataSource = [self defaulDataSource];
+        self.dataSource = [WPPHAssetDataSource sharedInstance];
```

Comment:
And keep in this this only applied has a default to allow the picker to work without extra configuration.